share/instruments/update-instruments-xml.py

```diff
@@ -0,0 +1,314 @@
+#!/usr/bin/env python3
```

Comment:
Why not integrate generateTs.py into this script? All information to generate instrumentsxml.h` is here. Except for the score ordering buy why not generating these files in a similar way?

Reply:
share/instruments/update-instruments-xml.py might call share/instruments/generateTs.py, but I'd keep it as separate script to be able to call it manually

Reply:
Calling generateTs.py is required when instruments.xml is edited, isn't it? But if this file is generated from a spreadsheet, editing instruments.xml should be discouraged to prevent a mismatch between the spreadsheet and the file. In that case it is never needed to run generateTs.py.
It must be very clear what will be the"real" source, instruments.xml or the spreadsheet. Otherwise we end up with two different sources and nobody knows what the real source is.

Reply:
True. Might still be useful to be able to run just the generateTs.py locally

Reply:
Not sure I agree with that, running generateTs.py might "encourage" people to make local changes to instruments.xml and that is what should discouraged.